.github/workflows/require-changeset.yaml

```diff
@@ -1,11 +1,11 @@
-name: Require Either a Changeset or a Label 'No Changeset Needed'
+name: Require Either a Changeset or a Label 'no changeset needed'
 on:
   pull_request:
     types: [opened, synchronize, reopened, labeled, unlabeled]
 
 jobs:
   require-changeset-or-label:
-    if: ${{ !contains(github.event.pull_request.labels.*.name, 'No Changeset Needed') && github.head_ref != 'changeset-release/main' }}
+    if: ${{ !contains(github.event.pull_request.labels.*.name, 'no changeset needed') && github.head_ref != 'changeset-release/main' }}
```

Comment:
The label present visually as "No Changeset Needed" can you verify this?

Reply:
I've just changed it to lowercase.